### PR TITLE
fixes panic

### PR DESCRIPTION
**Pull request by f41gh7**
https://github.com/VictoriaMetrics/helm-charts/issues/89

### CODE REVIEW

File: app/vmselect/prometheus/prometheus.go [hunk 1]
```diff
@@ -501,8 +501,12 @@ var deleteDuration = metrics.NewSummary(`vm_request_duration_seconds{path="/api/
 
 func resetRollupResultCaches() {
 	if len(*selectNodes) == 0 {
-		logger.Panicf("BUG: missing -selectNode flag")
+		logger.Warnf("missing -selectNode flag, cache reset request wont be propagated to the other select nodes." +
+			"you can fix it by adding flag with all vmselect node addresses from your cluster," +
+			" fox example: -selectNode select-addr-1:8481,select-addr-2:8481")
 	}
```
Comment (tenmozes) [inline app/vmselect/prometheus/prometheus.go:507]: should we return from func if no nodes specified?

Comment (f41gh7) [reply app/vmselect/prometheus/prometheus.go:511]: Yes, thought about it, changed.

File: app/vmselect/prometheus/prometheus.go [hunk 2]
```diff
@@ -500,8 +500,14 @@ func DeleteHandler(startTime time.Time, at *auth.Token, r *http.Request) error {
 var deleteDuration = metrics.NewSummary(`vm_request_duration_seconds{path="/api/v1/admin/tsdb/delete_series"}`)
 
 func resetRollupResultCaches() {
+	resetRollupResultCacheCalls.Inc()
+	// reset local cache.
```
Comment (valyala) [inline app/vmselect/prometheus/prometheus.go:504]:
```suggestion
	// Reset local cache before checking whether selectNodes list is empty.
	// This guarantees that at least local cache is reset if selectNodes list is empty.
```

**Review by valyala — APPROVED** [review]

**Comment by valyala** [comment]
Thanks!

**Pull request merged**